third_party/move/move-prover/lab/data/new-boogie-aptos-stdlib/current_boogie_3.fun_data

```diff
@@ -1,558 +1,564 @@
-smart_table::to_simple_map                        408           ok
-smart_table::unzip_entries                        413           ok
-smart_table::update_split_load_threshold          412           ok
-smart_table::update_target_bucket_size            412           ok
-smart_vector::contains                            734           ok
-smart_vector::index_of                            716           ok
-smart_vector::append                              413           ok
-smart_vector::borrow                              962           ok
-smart_vector::borrow_mut                         2533           ok
-smart_vector::destroy                             760           ok
-smart_vector::destroy_empty                       742           ok
-smart_vector::empty                               718           ok
-smart_vector::enumerate_mut                       414           ok
-smart_vector::enumerate_ref                       413           ok
-smart_vector::filter                              408           ok
-smart_vector::fold                                412           ok
-smart_vector::foldr                               413           ok
-smart_vector::for_each                            413           ok
-smart_vector::for_each_mut                        413           ok
-smart_vector::for_each_ref                        412           ok
-smart_vector::for_each_reverse                    412           ok
-smart_vector::is_empty                            716           ok
-smart_vector::length                              721           ok
-smart_vector::map                                 408           ok
-smart_vector::map_ref                             413           ok
-smart_vector::pop_back                          11800           ok
-smart_vector::push_back                           415           ok
-smart_vector::remove                              413           ok
-smart_vector::reverse                            1015           ok
-smart_vector::singleton                           970           ok
-smart_vector::swap                                408           ok
-smart_vector::swap_remove                         407           ok
-smart_vector::zip                                 412           ok
-smart_vector::zip_map                             414           ok
-smart_vector::zip_map_ref                         412           ok
-smart_vector::zip_mut                             412           ok
-smart_vector::zip_ref                             412           ok
-smart_vector::zip_reverse                         415           ok
-smart_vector::new                                 713           ok
-smart_vector::to_vector                           719           ok
-smart_vector::add_all                             810           ok
-smart_vector::clear                               741           ok
-smart_vector::empty_with_config                   712           ok
+# time  : 2024-08-31 02:00:57.796281 UTC
```

Comment:
Many of those individual timings look significantly worse?

Reply:
Last time, experiment was done on @junkil-park's machine so the number cannot be directly comparable.